docs/azure_jumpstart_arcbox/workbook/_index.md

```diff
@@ -0,0 +1,273 @@
+
+   ![Inventory Resource List](./inventory_resource_list.png)
+
+### Monitoring
+
+By enabling a resource in Azure Arc it gives you the ability to perform configuration management and monitoring tasks on those services as if they were first-class citizens in Azure. You are able to monitor your connected machine guest operating system performance and your Kubernetes clusters at the scope of the resource with VM and container Insights. In ArcBox the Azure Arc enabled servers and Azure Arc enabled Kubernetes clusters have been onboarded onto Azure Monitor.
+
+The "Monitoring" tab of the Jumpstart Workbook shows metrics and alerts for ArcBox resources organized in three sections:
+
+* *Alert Summary* - Shows an overview of alerts organized by severity and status. You can use the drop down menus to apply filters to the report. The following filters are available:
+  * Subscription: select one or multiple subscriptions in your environment to show available alerts.
+  * Resource Group: select one or more resource groups in your environment to show available alerts.
+  * Resource Type: select one or multiple resources types to show its alerts.
+  * Resources: select individual resources by name to visualize its alerts.
+  * Time Range: provide a time range in which the alert has been created.
+  * State: choose the alert type between New, Acknowledged or Closed.
+
+   ![Monitoring Alert Summary](./monitoring_alert_summary.png)
+
+* *Azure Arc Enabled Servers* - Shows metrics for CPU and memory usage on the Azure Arc enabled servers. Use the parameters section to select the select the Azure Arc enabled server as well as a time range to visualize the data.
+
+   ![Monitoring Azure Arc enabled Server Metrics](./monitoring_arc_servers.png)
+
+* *Azure Arc enabled Kubernetes* - Shows information and metrics from ArcBox's Azure Arc enabled Kubernetes clusters. Use the parameters section to filter data:
+  * Time Range: provide a time range for the metrics and logs to be displayed.
+  * Subscription: select your subscription where ArcBox is deployed.
+  * Log Analytics Workspace: select ArcBox's Log Analytics workspace.
+  * Azure Arcenabled K8s cluster: choose one of ArcBox's Azure Arc enabled Kubernetes clusters.
+  * Workload Type: choose one or multiple kubernetes deployment types.
+  * Namespace: choose one or multiple namespaces in the Kubernetes cluster.
+  * Workload Name: choose one of the deployments in your cluster.
+  * Pod Status: filter by Pod status like Pending/Running/Failed etc.
+  * Pod Name: filter by pod name in the namespace and workloadname selected.
+
+  With this report you will get several visualizations:
+
+  * *Pod and Container restart trend graphs.*
+
+     ![Monitoring Azure Arc enabled K8S Metrics](./monitoring_arc_kubernetes_1.png)
+
+  * *Pod count and status chart.*
+
+     ![Monitoring Azure Arc enabled K8S Metrics](./monitoring_arc_kubernetes_2.png)
```

Comment:
This is showing all the failed webhook containers, but I think its okay since it illustrates the graphs